Updated documentation for Google Chrome (#2)
mvllow/tailwindcss-safe-area: readme.md

| `min-h-screen-safe, h-screen-safe` | `calc(100vh - (env(safe-area-inset-top) + env(safe-area-inset-bottom)))` |
|                                    | `-webkit-fill-available`                                                 |

> Tip: To extend html content behind the safe area, set `viewport-fit=cover`

```html
<meta
	name="viewport"
	content="width=device-width, initial-scale=1.0, viewport-fit=cover"
/>
```

## Troubleshooting

> The `h-screen-safe` and `min-h-screen-safe` may not work as expected on Google Chrome. To make it work on this browser, you may need to add `-webkit-fill-available` on the `height` property to the all the div that wraps your content, up to the `html` div. To do so, simply add the following to your css file:

Reviewer: ```suggestion
The `h-screen-safe` and `min-h-screen-safe` may not work as expected on Google Chrome. Add `height: -webkit-fill-available` on parent nodes:
```
